c7n/resources/org.py

```diff
@@ -0,0 +1,361 @@
+# Copyright The Cloud Custodian Authors.
+# SPDX-License-Identifier: Apache-2.0
+
+from concurrent.futures import as_completed
+import logging
+import os
+import threading
+
+from botocore.exceptions import ClientError
+
+from c7n.credentials import assumed_session
+from c7n.executor import MainThreadExecutor
+from c7n.filters import Filter
+from c7n.query import QueryResourceManager, TypeInfo
+from c7n.resources.aws import AWS
+from c7n.utils import local_session, type_schema
+
+
+log = logging.getLogger("custodian.org-accounts")
+
+
+ORG_ACCOUNT_SESSION_NAME = "CustodianOrgAccount"
+
+
+def org_augment_tags(manager, resources):
+    client = local_session(manager.session_factory).client('organizations')
+    for r in resources:
+        r["Tags"] = client.list_tags_for_resource(
+            ResourceId=r["Id"]).get("Tags", [])
+    return resources
+
+
+class OrgAccess:
+
+    org_session = None
+
+    def parse_access_role(self):
+        params = {}
+        for q in self.data.get("query", ()):
+            params.update(q)
+        org_access = {
+            k: v for k, v in params.items() if k in ("org-access-role",)
+        }
+        return org_access.get('org-access-role')
```

Comment:
Unless I'm misreading something here this looks like it's doing more work than it needs to

```suggestion
        return params.get("org-access-role")
```